ta/mk/ta_dev_kit.mk

```diff
@@ -79,19 +82,19 @@ clean:
 	${q}if [ -d "$(O)" ]; then $(RMDIR) $(O); fi
 
 subdirs = .
-include  $(ta-dev-kit-dir)/mk/subdir.mk
+include  $(ta-dev-kit-dir$(sm))/mk/subdir.mk
 
-ifneq ($(binary),)
+ifneq ($(user-ta-uuid),)
```

Comment:
I don't get it. Where is `$(user-ta-uuid)` defined? Why replace `$(binary)` with `$(user-ta-uuid)` only here?

Reply:
I'll fix this in the rebase to follow